modules/learn/pages/clusters-and-availability/automatic-failover.adoc

```diff
@@ -35,14 +35,14 @@ An administrator-defined group of server-nodes within the cluster is unresponsiv
 
 Auto-failover is triggered:
 
-* Only on the occurrence of one event at a time.
-If multiple events are concurrent, auto-failover is not triggered.
-* Sequentially, only up to an administrator-specified maximum number of events.
+* Only on the occurrence of one failure at a time.
+If multiple nodes fail concurrently, auto-failover is not triggered.
```

Comment:
This is wrong. The whole auto-failover improvement is about multiple concurrent failures. I thought we had reviewed the changes already. So how did these 2 lines show up like this again? Did we lose the first set of changes?